templates/commands/upgrade/upgrade.go

```diff
@@ -149,15 +149,16 @@ func (c *Command) Run(ctx context.Context, args []string) error {
 	//
 	// TODO(upgrade):
 	//  - suggest diff / meld / vim commands?
-	fmt.Fprint(c.Stdout(), mergeInstructions+"\n")
+	fmt.Fprint(c.Stdout(), mergeInstructions+"\n\n")
```

Comment:
Is there a UT to check our stdout?

Reply:
Not yet, but there's a TODO for that a couple of lines above this.